src/app/(proper_react)/(redesign)/(authenticated)/admin/churn-subscribers/actions.tsx

```diff
@@ -9,17 +9,34 @@ import {
   deleteSubscriberChurns,
 } from "../../../../../../db/tables/subscriber_churns";
 import { SubscriberChurnRow } from "knex/types/tables";
+import { getServerSession } from "../../../../../functions/server/getServerSession";
+import { isAdmin } from "../../../../../api/utils/auth";
 
 export async function getAllChurns() {
+  const session = await getServerSession();
+  if (!session?.user?.email || !isAdmin(session.user.email)) {
+    return [];
```

Comment:
From the docs, I believe that returning `null` is the preferred way to do this currently:
https://nextjs.org/docs/app/building-your-application/authentication#server-actions

I found there is an experimental `forbidden` function that returns an `HTTP 403` which is an even better fit:
https://nextjs.org/docs/app/api-reference/functions/forbidden

I think we should return `null` for now, and maybe file a ticket in the tech debt backlog to switch to using this `forbidden` handler when it becomes stable. wdyt?